doc/jsk_perception/nodes/ssd_object_detector.md

```diff
@@ -0,0 +1,181 @@
+    image0002__labels.json  image0054.jpg            image0105.jpg           image0156__labels.json
+    image0003.jpg           image0054__labels.json   image0105__labels.json  image0157.jpg
+    image0003__labels.json  image0055.jpg            image0106.jpg           image0157__labels.json
+    ...
+    ```
+
+4. Train from the annotated dataset
+
+    ```bash
+    cd dataset/
+    rosrun jsk_perception ssd_train_dataset.py --gpu 0 dataset/label_names.yml dataset/
+    ```
+
+    When training process starts successfully, progress are shown on terminal as follows:
+
+    ```
+    Loaded 27 labels
+    train: 112, test: 28
+    epoch       iteration   lr          main/loss   main/loss/loc  main/loss/conf  validation/main/map
+    0           10          0.0001      9.9525      1.16335        8.78914                              
+    0           20          0.0001      6.11782     0.730058       5.38776                              
+    1           30          0.0001      4.26923     0.511913       3.75732                              
+    1           40          0.0001      4.01094     0.529855       3.48108                              
+    2           50          0.0001      3.1616      0.37409        2.78751                              
+    2           60          0.0001      2.80446     0.331363       2.4731                               
+    3           70          0.0001      2.57667     0.348          2.22867                              
+    3           80          0.0001      2.59318     0.350702       2.24248                              
+    4           90          0.0001      2.31911     0.3307         1.98841                              
+    4           100         0.0001      2.25847     0.285728       1.97274         0.737562             
+    4           110         0.0001      2.09784     0.265645       1.8322                               
+    5           120         0.0001      2.25548     0.309224       1.94626                              
+    5           130         0.0001      2.22602     0.311586       1.91443                              
+    6           140         0.0001      2.14396     0.332112       1.81185                              
+    6           150         0.0001      1.95564     0.274048       1.68159                              
+    7           160         0.0001      1.79976     0.273          1.52675                              
+    ...
+    ```
+
+    Weight files are generated on every 200 iterations by default. (e.g. `result/model_iter_400` as a weight file after 400 iterations)
+    You can monitor loss and stop training at anytime and resume with option `--resume`.
+
+    A generated weight file can be used as model file for `ssd_object_detector.py` by setting `~model_path` parameter as path to the weight file.
+
+## Reference
+
+```
+Wei Liu, et al. "SSD: Single shot multibox detector" ECCV 2016
+```
```

Comment:
How about adding two references to the documentation.

[ChainerCV SSD code](https://github.com/chainer/chainercv/tree/master/examples/ssd)
[Image Labelling tools](https://github.com/yuyu2172/image-labelling-tool/tree/master/examples/ssd)

Reply:
@yuyu2172 Sounds great! Updated docs :+1: